cmd/algorand-indexer/daemon.go

```diff
@@ -343,7 +344,14 @@ func makeOptions() (options api.ExtraOptions) {
 			fmt.Fprintf(os.Stderr, "failed to created disabled map config from file: %v", err)
 			panic(exit{1})
 		}
+
+		if len((*potentialDisabledMapConfig).Data) == 0 {
+			logger.Infof("NOTICE: All parameters are enabled since the provided parameter configuration file (%s) is empty.", suppliedAPIConfigFile)
+		}
+
 		options.DisabledMapConfig = potentialDisabledMapConfig
+	} else {
+		logger.Infof("NOTICE: Enable all parameters flag is set to: %v", enableAllParameters)
```

Comment:
In the log file, we can label the severity in a structured way.
```suggestion

		if len((*potentialDisabledMapConfig).Data) == 0 {
			logger.Warnf("All parameters are enabled since the provided parameter configuration file (%s) is empty.", suppliedAPIConfigFile)
		}

		options.DisabledMapConfig = potentialDisabledMapConfig
	} else {
		logger.Warnf("Enable all parameters flag is set to: %v", enableAllParameters)
```

Reply:
The second one is more of a informational one, but I can change the first one to a warning.